packages/core/test/payload-value-converter.spec.ts

```diff
@@ -150,7 +150,7 @@ describe('tsToEdm()', () => {
   });
 
   it('should parse BigNumber to Edm.Int64', () => {
```

Comment:
Is there a test for the reverse direction, i.e. `Edm.Int64` to `BigNumber`?

Reply:
Yes, but this didn't change: https://github.com/SAP/cloud-sdk/blob/3828e4661894fffbe618fd9f1f85560cea27c9c5/packages/core/test/payload-value-converter.spec.ts#L37